const getUser=async (req, res) => {
  try{
    


  }catch(erro){
    res.status(500).json({message: 'Server error'});
  }

}


module.exports= {getUser}